src/System.Net.Security/src/System/Net/_SslSessionsCache.cs

```diff
@@ -53,7 +53,7 @@ internal SslCredKey(byte[] thumbPrint, int allowedProtocols, bool serverMode, En
 
                 _HashCode ^= allowedProtocols;
                 _HashCode ^= (int)encryptionPolicy;
-                _HashCode ^= serverMode ? 5 : 7; //TODO (Issue #3362) used a prime number here as it's a XOR. Figure out appropriate value.
+                _HashCode ^= serverMode ? 5 : 7; // Used a prime number here as it's a XOR. The values should not conflict with EncryptionPolicy and other parts of the hash
```

Comment:
0 and 2 we cannot anyways use as these numbers already represent EncryptionPolicy ..
If we want to maintain the unique bits altogether ,, we can use the number beyond the sslprotocols values
i.e:
using 0x10000 and 0x20000 ..